cmd/web_acme.go

```diff
@@ -16,12 +19,12 @@ import (
 	"github.com/caddyserver/certmagic"
 )
 
-func runLetsEncrypt(listenAddr, domain, directory, email string, m http.Handler) error {
+func runACME(listenAddr string, m http.Handler) error {
 	// If HTTP Challenge enabled, needs to be serving on port 80. For TLSALPN needs 443.
 	// Due to docker port mapping this can't be checked programmatically
 	// TODO: these are placeholders until we add options for each in settings with appropriate warning
 	enableHTTPChallenge := true
-	enableTLSALPNChallenge := true
+	enableTLSALPNChallenge := false
```

Comment:
Generally LGTM. A question here, why is it changed to disabled?

Reply:
Because I don't see ALPN challenge being used, and this would disable the CA server from constantly trying to check it.

Reply:
Maybe we can leave a comment here for the reason. If not sure about the usage, it's better to keep it as before. Otherwise, we should mark this as a `breaking` behavior.

Reply:
Oh, sorry I was wrong, `ALPN` is being used correctly. I will change it back and correct the documentation

Reply:
Reverted, so no longer breaking in that aspect